docs/administration.rst

```diff
@@ -72,34 +72,38 @@ Basic Configuration
         SQLALCHEMY_DATABASE_URI = 'postgresql://<user>:<password>@<hostname>:5432/lemur'
 
 
-.. data:: SQLALCHEMY_POOL_SIZE
+.. data:: SQLALCHEMY_ENGINE_OPTIONS
```

Comment:
As this is sqlalchemy's config, is there any reference link that we could include to sqlalchemy's documentation for further info?

Reply:
good point, will add a link to the sqlalchemy's config page
https://flask-sqlalchemy.palletsprojects.com/en/2.x/config/